platforms/documentation/docs/src/docs/userguide/dep-man/understanding-dependency-resolution/dependency_caching.adoc

```diff
@@ -0,0 +1,265 @@
+However, where possible Gradle will check if the previously downloaded artifacts are valid before downloading again.
+This is done by comparing published SHA1 values in the repository with the SHA1 values for existing downloaded artifacts.
+
+- new versions of dynamic dependencies
+- new versions of changing modules (modules that use the same version string but can have different contents)
+
+Refreshing dependencies will cause Gradle to invalidate its listing caches.
+However:
+
+- it will perform HTTP HEAD requests on metadata files but _will not re-download them_ if they are identical
+- it will perform HTTP HEAD requests on artifact files but _will not re-download them_ if they are identical
+
+In other words, refreshing dependencies _only_ has an impact if you actually use dynamic dependencies _or_ that you have changing dependencies that you were not aware of (in which case it is your responsibility to declare them correctly to Gradle as changing dependencies).
+
+It's a common misconception to think that using `--refresh-dependencies` will force the download of dependencies.
+This is **not** the case: Gradle will only perform what is strictly required to refresh the dynamic dependencies.
+This _may_ involve downloading new listings, metadata files, or even artifacts, but the impact is minimal if nothing changed.
+
+[[sub:ephemeral-ci-cache]]
+== Dealing with ephemeral builds
+
+It's a common practice to run builds in ephemeral containers.
+A container is typically spawned to only execute a single build before it is destroyed.
+This can become a practical problem when a build depends on a lot of dependencies which each container has to re-download.
+To help with this scenario, Gradle provides a couple of options:
+
+- <<sub:cache_copy,copying the dependency cache>> into each container
+- <<sub:shared-readonly-cache,sharing a read-only dependency cache>> between multiple containers
+
+[[sub:cache_copy]]
+=== Copying and reusing the cache
+
+The dependency cache, both the file and metadata parts, are fully encoded using relative paths.
+This means that it is perfectly possible to copy a cache around and see Gradle benefit from it.
+
+The path that can be copied is `$GRADLE_USER_HOME/caches/modules-<version>`.
+The only constraint is placing it using the same structure at the destination, where the value of `GRADLE_USER_HOME` can be different.
+
+Do not copy the `*.lock` or `gc.properties` files if they exist.
+
+Note that creating the cache and consuming it should be done using compatible Gradle version, as shown in the table below.
+Otherwise, the build might still require some interactions with remote repositories to complete missing information, which might be available in a different version.
+If multiple incompatible Gradle versions are in play, all should be used when seeding the cache.
+
+[%header%autowidth,compact]
```

Comment:
I question if we want to include this table. Having another thing to update on a new release would be painful, and this will need an update soon as I have `metadata-2.107` on my machine.

Reply:
acceptable for now.